src/demos/MultiPoint.vue

```diff
@@ -51,6 +51,230 @@
       </ol-source-vector>
     </ol-vector-layer>
   </ol-map>
+
+  <ol-map
+    :loadTilesWhileAnimating="true"
+    :loadTilesWhileInteracting="true"
+    style="height: 400px"
+  >
+    <ol-view
+      ref="view"
+      :center="center2"
+      :zoom="zoom2"
+      :projection="projection2"
+    />
+
+    <ol-tile-layer>
+      <ol-source-osm />
+    </ol-tile-layer>
+
+    <ol-vector-layer>
+      <ol-source-vector>
+        <ol-feature>
+          <ol-geom-polygon
```

Comment:
since the demo shows a polygon, the demos should be moved to the Demo for polygons. And I would like to have a separate small headline for the gadients describing how to use them.